packages/extension-dapp/src/index.ts

```diff
@@ -63,35 +70,42 @@ export function web3Enable (originName: string): Promise<InjectedExtension[]> {
     throw new Error('You must pass a name for your app to the web3Enable function');
   }
 
-  web3EnablePromise = documentReadyPromise((): Promise<InjectedExtension[]> =>
-    getWindowExtensions(originName)
-      .then((values): InjectedExtension[] =>
-        values
-          .filter((value): value is [InjectedExtensionInfo, Injected] => !!value[1])
-          .map(([info, ext]): InjectedExtension => {
-            // if we don't have an accounts subscriber, add a single-shot version
-            if (!ext.accounts.subscribe) {
-              ext.accounts.subscribe = (cb: (accounts: InjectedAccount[]) => void | Promise<void>): Unsubcall => {
-                ext.accounts.get().then(cb).catch(console.error);
-
-                return (): void => {
-                  // no ubsubscribe needed, this is a single-shot
-                };
-              };
-            }
-
-            return { ...info, ...ext };
+  web3EnablePromise = documentReadyPromise(
+    (): Promise<InjectedExtension[]> =>
```

Comment:
Like the previous comment, quite difficult to extract readability improvements vs logic changes. It seems the same.

Reply:
Is there an easy way to revert the formating? Shouldn't `yarn run lint` do the trick? Thats what I ran on this repo